### PR TITLE
Update ontology.ttl after merge of 30/05

### PR DESCRIPTION
**Pull request by Certiman**
Mainly definition TODOs and the appendixD2Index property.

### CODE REVIEW

**Review by gatemezing — APPROVED** [review]

File: ontology.ttl [hunk 1]
```diff
@@ -4922,7 +4926,7 @@ era:length rdf:type owl:DatatypeProperty ;
                        "IPP_Length" ,
                        "ITU_Length" ,
                        "SOLLength" ;
-           era:appendixD2Index "2.3.6" ;
+           era:appendixD2Index "2.3.6" , "3.2.3" , "3.2.4", ;
```
Comment (gatemezing) [inline ontology.ttl:4929]: There is an error in this line. It should be something like this `era:appendixD2Index "2.3.6" , "3.2.3" , "3.2.4" ;`

**Review by gatemezing — COMMENTED** [review]
@Certiman , please fix the issue in Line 4929, with the triple 
`era:appendixD2Index "2.3.6" , "3.2.3" , "3.2.4" ;` and check if no error when loading with Protégé.

File: ontology.ttl (hunk 1 above)
Comment (Certiman) [reply ontology.ttl:4929]: That is indeed correct

**Comment by Certiman** [comment]
Typo corrected

**Pull request merged**